microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

```diff
@@ -41,7 +41,7 @@ We are aware that Chromium has revised the timelines for Manifest V2 sunset. We
 | TBD - See note above | Microsoft Partner Center will no longer accept updates to existing Manifest V2 extensions. Developers can submit updates for migrating a V2 extension to V3. | Microsoft Edge stops running Manifest V2 extensions. Enterprises can allow Manifest V2 extensions to run on Microsoft Edge using Enterprise policies. |
 | TBD - See note above | No change. | Manifest V2 extensions will no longer function in Microsoft Edge even with the use of Enterprise policies. |
 
-Manifest V2 extensions will continue to be supported through Enterprise policies at least until Chromium Manifest V2 support timeline (which is currently January 2024). 
+Manifest V2 extensions will continue to be supported through Enterprise policies at least until Chromium Manifest V2 support timeline (which is currently June 2024). 
```

Comment:
```suggestion
Manifest V2 extensions will continue to be supported through Enterprise policies at least until a year after Chromium Manifest V2 support timeline.  Per the Chromium Manifest V2 support timeline, Manifest V2 extensions will be disabled as early as June 2024.
```
tech review this Suggestion.  line 44 specifies in particular "Enterprise".  @captainbrosset - could you pls check this wording against:
https://developer.chrome.com/blog/resuming-the-transition-to-mv3#the_phase-out_timeline
which reads: 
We will begin disabling Manifest V2 extensions in pre-stable versions of Chrome (Dev, Canary, and Beta) as early as June 2024,
..
Enterprises using the ExtensionManifestV2Availability policy to ensure the continued functioning of Manifest V2 extensions in their organization will have one additional year - until June 2025 - to migrate the Manifest V2 extensions in their organization.

I was looking for Siv's username, to review this, and I lost v1 of this comment, so asking you.

Reply:
live article has a link to a timeline:

Meanwhile, please refer to the [Chromium timelines](https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/) for planning your extension's migration.

"Manifest V2 support timeline"
https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/ 

Two problems - I should push a commit to fix them:
* Link text must match destination title, else confusing nav.
* per style guide, usually omit "please", w/ rare exceptions.

Reply:
pushed commit that's a rewrite of article